### ISSUE TITLE
Scam

### ISSUE THREAD

**Issue by roimee6** [NopeCHALLC/nopecha-extension#66]
Nopecha is a scam, don't buy anything from them, apis never give tokens to solve your captcha

The api is present but you'll never get anything to solve your captcha in addition, no support available, the staff never reply

**Comment by KyrylRohovyk** [comment]
Agree. I've bought a basic subscription and it doesn't work. Can't reach their support.

**Comment by raghu-nandan-bs** [comment]
Basic subscription *WAS* working earlier, now its completely trash. I feel cheated of my money.

**Comment by zaheersani** [comment]
We purchased Enterprise subscription ($99), which says, 24*7 email and ticket support. They don't respond anywhere. And we tried to request a refund but we did not receive any response from them. We thought that Enterprise would be able to solve Captcha faster than Free plan, but there is no difference in the speed.
